core/spec.md

```diff
@@ -1278,21 +1336,25 @@ The following describes the attributes of Registry model:
   - This contains the list of attributes defined as part of a nested resource
   - Type: Object, see `attributes` above
   - MAY be present when the owning attribute's `type` is `object`, otherwise it
-    MUST NOT be present. It MAY be absent or an empty list if there are not
+    MUST NOT be present. It MAY be absent or an empty list if there are no
     defined attributes for the nested `object`
+
 - `attributes."STRING".item`
-  - Defines the nested resources that this attribute references.
-    This attribute MUST only be used when the owning attribute's `type` value
-    is `map` or `array`
+  - Defines the nested resources that this attribute references.  This
```

Comment:
Nit: double space. (We're also pretty inconsistent about whether or not these list items end in a period. I haven't noted that in comments...)

Reply:
I've been trying to leave them off, but as you've noticed... I can't stop my fingers from introducing typos :-)